### DISCUSSION TITLE
Active vs passive I/O filters

### DISCUSSION THREAD

**Discussion by willwillems** [newdigate/teensy-eurorack#29] [General]
Yo @newdigate!

Super rad project, I'm designing a custom board to retrofit a Xone 3D, I'm looking at the CS42448's datasheet and am deciding between the active and passive IO filters, the passive ones seem *much* simpler but from the docs the practical difference is kinda hard to gauge, have you ever compared them or do you have any complaints about current audio quality with the passive ones?

Also does your CS42448 run very hot?

Thanks!

**Comment by newdigate** [comment]
Hey @willwillems 
Thanks!!! Glad you find it useful :) 

 Only thing about passive input filters which I don't like is that I can't connect my guitar as the signal is too quiet. Other than that, passive filters sound perfectly good on input and output. I've never compared passive to active, but I do like the idea of plugging my guitar in one day, so I might just upgrade to using pre-amps / active filters :) 

Yes - that cs42448 runs very hot - I used to use tiny little heatsinks, but after a while I gave up and I've never has any issue with that codec - it just works!!!
